### PR TITLE
cmd: add info command

### PR DESCRIPTION
**Pull request by SMillerDev**
Current output
```
===
✅ - Php (homebrew.mxcl.php)
 Loaded: true
 Active: true
 User: sean
 PID: 90734
-
Command: /bin/cmd
Working directory:
Root directory:
Log:
Error log:
File:           /usr/local/opt/php/homebrew.mxcl.php.plist ✅
===
```

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
@SMillerDev Can you explain the motivation for this? Just to warn you: I'm gonna have pretty strong opinions on what the output for this should look like depending on the use-case.

**Comment by SMillerDev** [comment]
Basically `brew services` does not have information like `systemctl status X` would have. This would IMHO improve over the fairly cluttered `brew ls` output where `user` and `file` seem like overkill.

Having information like the log location without reading the formula would also be useful for people who are surprised that their service is erroring on startup (like after ever postgres major version).

> 'm gonna have pretty strong opinions on what the output for this should look like 

That's okay, I have no strong opinions about the output so you can take the lead there.

**Comment by MikeMcQuaid** [comment]
> Having information like the log location without reading the formula would also be useful for people who are surprised that their service is erroring on startup (like after ever postgres major version).

👍🏻. Do you want this to be machine readable, human readable or both?

**Comment by SMillerDev** [comment]
I'm mostly aiming for human readable now. If we ever want machine readable we can make it JSON

File: lib/service/commands/info.rb
Comment (MikeMcQuaid) [inline lib/service/commands/info.rb:14]:
```suggestion
            puts unless index.zero?
```

File: lib/service/commands/info.rb
Comment (MikeMcQuaid) [inline lib/service/commands/info.rb:16]:
```suggestion
```

File: lib/service/commands/info.rb
Comment (MikeMcQuaid) [inline lib/service/commands/info.rb:20]:
```suggestion
      def pretty_bool(bool)
```

File: lib/service/commands/info.rb
Comment (MikeMcQuaid) [inline lib/service/commands/info.rb:25]:
```suggestion
        puts "#{Tty.bold}#{service.name}#{Tty.reset} (#{service.service_name})\n"
        puts "Running: #{pretty_bool(service.pid?)}"
```

File: lib/service/commands/info.rb
Comment (MikeMcQuaid) [inline lib/service/commands/info.rb:27]:
```suggestion
        puts " #{Tty.bold}Loaded:#{Tty.reset} #{pretty_bool(service.loaded?)}"
        puts " #{Tty.bold}Active:#{Tty.reset} #{pretty_bool(service.pid?)}"
```

File: lib/service/commands/info.rb
Comment (MikeMcQuaid) [inline lib/service/commands/info.rb:39]:
```suggestion
        puts "#{Tty.bold}File:#{Tty.reset} #{service.service_file} #{print_bool(service.service_file_present?)}"
```

File: lib/service/commands/info.rb
Comment (MikeMcQuaid) [inline lib/service/commands/info.rb:28]:
```suggestion
        puts " #{Tty.bold}User:#{Tty.reset} #{System.user_of_process(service.pid)}" if service.pid?
```

File: lib/service/commands/info.rb
Comment (MikeMcQuaid) [inline lib/service/commands/info.rb:33]:
```suggestion
```

**Comment by SMillerDev** [comment]
Removing the draft status, but this will need to be rebased on https://github.com/Homebrew/homebrew-services/pull/411

**Comment by MikeMcQuaid** [comment]
@SMillerDev can you dump the latest JSON/non-JSON output here for review?

**Comment by SMillerDev** [comment]
brew services info php
```
php (homebrew.mxcl.php)
Running: ✅
Loaded: ✅
Schedulable: ❌
User: sean
PID: 2533
```
brew services info php --verbose
```
php (homebrew.mxcl.php)
Running: ✅
Loaded: ✅
Schedulable: ❌
User: sean
PID: 2533
File: /Users/sean/Library/LaunchAgents/homebrew.mxcl.php.plist ✅
Command: /usr/local/opt/php/sbin/php-fpm --nodaemonize
Working directory: /usr/local/var
Error log: /usr/local/var/log/php-fpm.log
```
brew services info php --json
```json
[
  {
    "name": "php",
    "service_name": "homebrew.mxcl.php",
    "running": true,
    "loaded": true,
    "schedulable": false,
    "pid": 2533,
    "exit_code": 0,
    "user": "sean",
    "status": "started",
    "file": "/Users/sean/Library/LaunchAgents/homebrew.mxcl.php.plist",
    "command": "/usr/local/opt/php/sbin/php-fpm --nodaemonize",
    "working_dir": "/usr/local/var",
    "root_dir": null,
    "log_path": null,
    "error_log_path": "/usr/local/var/log/php-fpm.log",
    "interval": null,
    "cron": null
  }
]
```

File: lib/service/commands/info.rb
Comment (MikeMcQuaid) [inline lib/service/commands/info.rb:26]:
```suggestion
        return bool if !$stdout.tty? || Homebrew::EnvConfig.no_emoji?
        
        if bool
          "#{Tty.bold}#{Formatter.success("✔")}#{Tty.reset}"
        else
           "#{Tty.bold}#{Formatter.error("✘")}#{Tty.reset}"
        end
```

for consistency with `pretty_installed` and `pretty_uninstalled` used by `brew info` in Homebrew/brew. Might be nice in follow-up PRs to refactor those functions so they can be used here.

**Review by MikeMcQuaid — APPROVED** [review]
Looks good. Once style suggestion.

File: lib/service/commands/info.rb
Comment (SMillerDev) [reply lib/service/commands/info.rb]: > Might be nice in follow-up PRs to refactor those functions so they can be used here.

Sounds good, maybe I'll even wait until that PR is released before I merge the service changes 😅

**Comment by MikeMcQuaid** [comment]
> @MikeMcQuaid can you help me ensure it's always a TTY in Rspec?

@SMillerDev Do you mean mocking it? `allow_any_instance_of(IO).to receive(:tty?).and_return(true)` will do it, I think. Stick that either in the relevant test or in a `before` block for multiple tests.

**Pull request merged**
